pkg/util/coverage/coverage.go

```diff
@@ -59,7 +59,9 @@ func InitCoverage(name string) {
 	}
 
 	// Set up the unit test framework with the required arguments to activate test coverage.
-	flag.CommandLine.Parse([]string{"-test.coverprofile", tempCoveragePath()})
+	if err := flag.CommandLine.Parse([]string{"-test.coverprofile", tempCoveragePath()}); err != nil {
+		panic(err)
```

Comment:
Suggestion: log some additional info here so we know where we are i.e 'Failed to prepare coverage framework..'

Reply:
This function seems to use `fmt.Println` logging, so added a line before `panic()`ing